plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/PipelineStore.java

```diff
@@ -69,6 +91,7 @@ public PipelineStore(Settings settings, ThreadPool threadPool, Environment envir
 
     @Override
     protected void doStart() {
+        client = injector.getInstance(Client.class);
```

Comment:
wonder if this is the best way to get a client, can't come up with better ones though at the moment....